src/internal/kopia/wrapper.go

```diff
@@ -515,3 +518,118 @@ func isErrEntryNotFound(err error) bool {
 	return strings.Contains(err.Error(), "entry not found") &&
 		!strings.Contains(err.Error(), "parent is not a directory")
 }
+
+func (w Wrapper) Maintenance(
+	ctx context.Context,
+	safety control.Safety,
+	quickMaintenance, force bool,
```

Comment:
Feels like all three of these could get rolled into some kind of `control.RepoMaintenance`